docs/documentation/development/bindings/faq.md

```diff
@@ -36,3 +36,9 @@ Here is a list of frequently asked questions around the development of bindings.
 5. _When creating a Thing through my ThingHanderFactory, does it exactly have to have the channels that are defined in the thing type description?_
  
     It must at least have the channels that are defined in its thing type (and they are already automatically added by the super() implementation). Nonetheless, you are free to add any number of additional channels to the thing.
+
+### Presenting device data to a user
+
+1. I have an image in my binding and want to pass it to the framework to be shown in UIs. What is the best way to achieve this?
```

Comment:
Remove "to be shown in UIs" - you might want to have it sent by e-mail or be persisted in a database; it simply does not matter.